webextensions/manifest/theme.json

```diff
@@ -64,7 +64,8 @@
                   }
                 ],
                 "firefox_android": {
-                  "version_added": false
+                  "version_added": "65",
+                  "notes": "Unlike Firefox for desktop, this value is still mandatory for Firefox for Android."
```

Comment:
I think it's better to avoid contrasts with other specific browsers in notes. Also, pedantically, I think "property" is better than "value" here. So just "This property is mandatory." would be better here.

Reply:
```suggestion
	                  "notes": "This property is mandatory."
```

Reply:
Change made